content/python/concepts/built-in-functions/terms/filter/filter.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'filter()'
+Description: 'A built-in function that applies a function to each item in an iterable and returns the values that are true.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+'A built-in function that applies a function to each item in an interable and returns the values that are true. The output of `filter()` is a filter object, in the examples below `list()` is applied to the filter expressions to return a new list.'
+
+## Syntax
+
+```py
+filter(func_name, iterable)
+```
+
+## Example 1
+
+Use `filter()` to return all the odd values in the following list:
+
+```codebyte/python
+nums = [33, 99, 63, 29, 25, 96, 61, 25, 22, 89, 90, 90]
+
+print(list(filter(lambda x: x % 2 != 0, nums)))
+
+# Output: [33, 99, 63, 29, 25, 61, 25, 89]```
+
+## Example 2
+
+Use `filter()` to return only the strings that are all lowercase:
```

Comment:
```suggestion
`filter()` can be used to return only the strings that are all lowercase:
```

Reply:
I imagine the standards have evolved since the start of docs, but early on examples were usually framed as an imperative like this - https://www.codecademy.com/resources/docs/javascript/strings/replace

Has there been a move away from this approach?

Reply:
Yeah I think if I recall correctly from prior discussions, we want to move away from any/all usage of "we" and "you" pronouns (even ones implied with imperatives) to make the Docs entries more manual-like.

Reply:
That is to say, the entry you cited should also be edited at some point in the future with the imperatives removed and replaced.